gandalf.js

```diff
@@ -2,76 +2,77 @@
  * Extras Plugin
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.